### DISCUSSION TITLE
Don't really know where to start

### DISCUSSION THREAD

**Discussion by rvanlieshout** [minvws/nl-kat-coordination#450] [Q&A]
Hi all. Is there somebody able and willing to help me out for a minute? I've got no clue where to start here. I've got OpenKAT running but seem unable to add a website or anything that should be scanned

I'm signed in as a red team user and and able to manually create objects, but am unable to create all required fields

**Comment by underdarknl** [comment]
Dear @rvanlieshout 
Yes we can!, tomorrow at 15:00 we have our weakly question and answer meeting.Would you be available to join us then?
A good startgin point would be to Add a url object, from where KAT will deduct the website etc automatically. If you then also set a clearance level on this url, this should propagate onto other objects and start related scans based on what tools are enabled in your katalogus.
Did the onboarding process not take you trough these steps?

**Reply by underdarknl** [reply]
The meet is online at https://meet.hack42.nl/kattendemo at 15:00 Amsterdam time.

**Reply by rvanlieshout** [reply]
Awesome. I'll join tomorrow

> Did the onboarding process not take you trough these steps?

Yes it did. It started with a nice selection of a report and some questions. After that I'm only able to manually add objects, but was unable to do so because of a missing NetLoc of errors

**Comment by rvanlieshout** [comment]
It turned out I was using mixing the dev setup with the production setup. 

Case closed
